server/routerlicious/packages/protocol-base/src/protocol.ts

```diff
@@ -46,6 +49,9 @@ export function isSystemMessage(message: ISequencedDocumentMessage) {
 	}
 }
 
+/**
+ * @deprecated moved to `@fluidframework/container-loader#protocol.ts`
+ */
 export interface ILocalSequencedClient extends ISequencedClient {
```

Comment:
Prob same for this one as Alex's comment above - I'd expect that container-loader is the only usage.